learn/advanced/asynchronous_operations.md

```diff
@@ -252,70 +197,19 @@ This command returns tasks two at a time, starting from task `uid` `10`.
 ```json
 {
   "results": [
-    {
-      "uid": 10,
-      "indexUid": "elements",
-      "status": "succeeded",
-      "type": "indexCreation",
-      "canceledBy": null,
-      "details": {
-        "primaryKey": null
-      },
-      "error": null,
-      "duration": "PT0.006034S",
-      "enqueuedAt": "2022-06-20T13:41:42.446908Z",
-      "startedAt": "2022-06-20T13:41:42.447477Z",
-      "finishedAt": "2022-06-20T13:41:42.453511Z"
-    },
-    {
-      "uid": 9,
-      "indexUid": "particles",
-      "status": "succeeded",
-      "type": "indexCreation",
-      "canceledBy": null,
-      "details": {
-        "primaryKey": null
-      },
-      "error": null,
-      "duration": "PT0.007317S",
-      "enqueuedAt": "2022-06-20T13:41:31.841575Z",
-      "startedAt": "2022-06-20T13:41:31.842116Z",
-      "finishedAt": "2022-06-20T13:41:31.849433Z"
-    }
+    …
   ],
   "limit": 2,
   "from": 10,
   "next": 8
 }
 ```
 
-To view the next page of results, you would repeat the same query, replacing the value of `from` with the value of `next`:
+To view the next set of results, you would repeat the same query, replacing the value of `from` with the value of `next`:
 
 <CodeSamples id="get_all_tasks_paginating_2" />
 
-When the returned value of `next` is `null`, you have reached the final page of results.
-
-## Task workflow
```

Comment:
We should keep this too 👀 
I would suggest:
- We move 51-59 to the API reference under `/reference/api/tasks.md#status`. Maybe add a table for this
- Move the task workflow to line 51